### DISCUSSION TITLE
ImportError: DLL load failed while importing _imaging: The specified module could not be found.

### DISCUSSION THREAD

**Discussion by robintakah** [mrdbourke/zero-to-mastery-ml#97] [General]
Greetings,
Please help.

I have loaded conda create --prefix ./env pandas numpy matplotlib scikit-learn.
I have also installed the jupyter notebook.
However, when I try to import pandas, numpy matplotlib and scilearn, I receive an import error as shown below.

Please assist. I am unsure of the cause of this

* Course video/timestamp: (e.g. video number 42 on ZTM, timestamp 18:14)

## My error
TODO: I am getting an error when I run my code in jupyter notebook
## My code
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import scilearn as sklearn

TODO: Add your code here, best to format with backticks as well, for example: 
ImportError                               Traceback (most recent call last)
Cell In[8], line 3
      1 import pandas as pd
      2 import numpy as np
----> 3 import matplotlib.pyplot as plt
      4 import scilearn as sklearn

File ~\sample_project_1\env\Lib\site-packages\matplotlib\__init__.py:161
    157 from packaging.version import parse as parse_version
    159 # cbook must import matplotlib only within function
    160 # definitions, so it is safe to import from it here.
--> 161 from . import _api, _version, cbook, _docstring, rcsetup
    162 from matplotlib.cbook import sanitize_sequence
    163 from matplotlib._api import MatplotlibDeprecationWarning

File ~\sample_project_1\env\Lib\site-packages\matplotlib\rcsetup.py:27
     25 from matplotlib import _api, cbook
     26 from matplotlib.cbook import ls_mapper
---> 27 from matplotlib.colors import Colormap, is_color_like
     28 from matplotlib._fontconfig_pattern import parse_fontconfig_pattern
     29 from matplotlib._enums import JoinStyle, CapStyle

File ~\sample_project_1\env\Lib\site-packages\matplotlib\colors.py:52
     49 from numbers import Real
     50 import re
---> 52 from PIL import Image
     53 from PIL.PngImagePlugin import PngInfo
     55 import matplotlib as mpl

File ~\sample_project_1\env\Lib\site-packages\PIL\Image.py:88
     79 MAX_IMAGE_PIXELS: int | None = int(1024 * 1024 * 1024 // 4 // 3)
     82 try:
     83     # If the _imaging C module is not present, Pillow will not load.
     84     # Note that other modules should not refer to _imaging directly;
     85     # import Image and use the Image.core variable instead.
     86     # Also note that Image.core is not a publicly documented interface,
     87     # and should be considered private and subject to change.
---> 88     from . import _imaging as core
     90     if __version__ != getattr(core, "PILLOW_VERSION", None):
     91         msg = (
     92             "The _imaging extension was built for another version of Pillow or PIL:\n"
     93             f"Core version: {getattr(core, 'PILLOW_VERSION', None)}\n"
     94             f"Pillow version: {__version__}"
     95         )

ImportError: DLL load failed while importing _imaging: The specified module could not be found.

**Comment by kRiShNa-429407** [comment]
Please check the thread on the discord server covering this issue.
